src/EnergyPlus/MixedAir.cc

```diff
@@ -275,8 +275,9 @@ namespace MixedAir {
 	// use these. They are cleared by clear_state() for use by unit tests, but normal simulations should be unaffected.
 	// This is purposefully in an anonymous namespace so nothing outside this implementation file can use it.
 		bool InitOAControllerOneTimeFlag( true );
-		bool InitOAControllerSetPointCheckFlag( true );
+		Array1D_bool InitOAControllerSetPointCheckFlag( true );
```

Comment:
Hmm, it's not a big deal since it gets deallocated and reallocated, but I am curious what the `true` argument does here.  Maybe creates a bool array with size 1 with an initial value of true?